hackathon_site/event/jinja2/event/dashboard_base.html

```diff
@@ -32,14 +32,36 @@ <h2 class="formH2">Complete your application</h2>
             {% elif application is not none and review is not none %}
                 <br />
                 {% if review.status == "Accepted" %}
-                    <h4 class="formH2"><b>Congratulations!</b> You've been accepted into {{ hackathon_name }}!</h4>
-                    <p><strong>Status</strong>: {{ status }}</p>
+                    {% if application.rsvp == True %}
```

Comment:
Because python reasons, these should be `if application.rsvp is True` (see [here](https://stackoverflow.com/questions/27276610/boolean-identity-true-vs-is-true)). Same goes when comparing with `None` and `False`.